R/RT_functions.R

```diff
@@ -37,16 +37,6 @@ create_ticket <- function(award, requestor) {
                                  subject = subject,
                                  rt_base = 'https://support.nceas.ucsb.edu/rt')
 
-   # if (!grepl('created', httr::content(ticket))) {
```

Comment:
Can you explain in your commit what the RT package isn't supporting?

Reply:
fixed to support the new changes in RT